### PR TITLE
Add nava `play` library

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### What does this implement/fix? Explain your changes.
Formerly we used `play_sound` function to play sound in mytimer. Now we add `nava`'s `play` function to play sounds.

### CODE REVIEW

**Comment by sadrasabouri** [comment]
Should we separate runs for each OS and install dependency packages for each repositories in which `nava` is being used?
That's not a good think. @sepandhaghighi

**Comment by sadrasabouri** [comment]
@sepandhaghighi do we need to install `sudo apt-get install alsa-utils` for solving this issue?
That would be a mess tbh.

File: dev-requirements.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ setuptools>=40.8.0
 vulture>=1.0
 bandit>=1.5.1
 pydocstyle>=3.0.0
-
+nava==0.3
```
Comment (sepandhaghighi) [inline dev-requirements.txt:6]: Please move this line to the top of the file.

File: mytimer/params.py [hunk 2]
```diff
@@ -4,7 +4,6 @@
 MY_TIMER_VERSION = "1.2"
 ADDITIONAL_INFO = "Additional information: Press `Ctrl+C` to exit."
 INPUT_ERROR_MESSAGE = "[Error] Wrong input"
-SOUND_ERROR_MESSAGE = "[Error] Unable to play sound"
```
Comment (sepandhaghighi) [inline mytimer/params.py:7]: Why did you delete this line?
This program is a CLI and should not raise an error in this particular scenario.

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Please take a look at my comments 🔥
Additionally, kindly update the `CHANGELOG.md` file.

**Comment by sepandhaghighi** [comment]
> @sepandhaghighi do we need to install `sudo apt-get install alsa-utils` for solving this issue? That would be a mess tbh.

No, it is out of the scope of this app. 
We should cover the `play` function with a `try-except` block. 
Something like:

```python
def play_sound(sound_path):
    """
    Play sound.

    :param sound_path: sound path
    :type sound_path: str
    :return: None
    """
    try:
        play(sound_path)
    except Exception:
        print(SOUND_ERROR_MESSAGE)
```

**Comment by sadrasabouri** [comment]
Thanks for the review @sepandhaghighi 
Your concerns addressed in commits.
Please re-review.

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - 1 new program
 	1. `animedoro`
 ### Changed
+- Sound playing system updated
+- Python 3.5 dropped
 - `short-break` program duration changed from `10 minutes` to `5 minutes`
 - `long-break` program duration changed from `30 minutes` to `15 minutes`
 - `pomodoro` program updated
```
Comment (sepandhaghighi) [inline CHANGELOG.md:16]: It is crucial to mention the addition of a new dependency. 
- `nava` added to `requirements.txt`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (sadrasabouri) [reply CHANGELOG.md:17]: fixed in [38f4a1f](https://github.com/sepandhaghighi/mytimer/pull/86/commits/38f4a1f2f12e4618e78b4b81ea4d904d71f8bddf).

**Review by sepandhaghighi — APPROVED** [review]
🚀
